src/Raven.Server/Smuggler/Documents/Data/SmugglerResult.cs

```diff
@@ -21,7 +21,8 @@ public SmugglerResult()
 
             Documents = new CountsWithSkippedCountAndLastEtag();
             RevisionDocuments = new CountsWithLastEtag();
-            Identities = new Counts();
+            LocalIdentities = new Counts();
```

Comment:
If I refactor LocalIdentities to have the name of Identities as before, it should prevent studio break, no?